compute_tools/src/sql/pre_drop_role_revoke_privileges.sql

```diff
@@ -15,12 +18,30 @@ BEGIN
         -- ii) it's easy to add more schemas to the list if needed.
         WHERE schema_name IN ('public')
     LOOP
+        SET LOCAL ROLE neon_superuser;
+
         revoke_query := format(
-            'REVOKE ALL PRIVILEGES ON ALL TABLES IN SCHEMA %I FROM {role_name} GRANTED BY neon_superuser;',
+            'REVOKE ALL PRIVILEGES ON ALL TABLES IN SCHEMA %I FROM {role_name} GRANTED BY neon_superuser',
             schema
         );
 
         EXECUTE revoke_query;
```

Comment:
It is not at all clear to me why I need to keep this statement around, but if I don't, `test_compute_drop_role` will fail. Help in diagnosing this is appreciated.

Reply:
Keeping this statement around tells me there is probably something more to look at than just `information_schema.role_table_grants`.